app/email.py

```diff
@@ -1,21 +1,16 @@
-from threading import Thread
-
-from flask import current_app, render_template
+import os
+from flask import render_template
 from flask.ext.mail import Message
 from . import mail
+from app import create_app
 
 
-def send_async_email(app, msg):
+def send_email(recipient, subject, template, **kwargs):
```

Comment:
Can you edit the `send_email` function to add to the task queue? That way, we dont have to edit all the projects using `send_email` to take advantage of the task queue addition. i.e. all calls in the form `send_email(recipient, subject, template, **kwargs)` should add to the task queue. One way to do this is the following (not sure about validity):
```
def send_email(recipient, subject, template, **kwargs):
  get_queue().enqeue(send_email_handler, recipient=recipient, subject=subject, template=template, **kwargs)

def send_email_handler(recipient,subject, template, **kwargs): 
      app = create_app(os.getenv('FLASK_CONFIG') or 'default')
      with app.app_context():
         msg = Message(app.config['EMAIL_SUBJECT_PREFIX'] + ' ' + subject,
                       sender=app.config['EMAIL_SENDER'],
                       recipients=[recipient])
         msg.body = render_template(template + '.txt', **kwargs)
         msg.html = render_template(template + '.html', **kwargs)
         mail.send(msg)
```
Main aim is so all the apps dont have to edit their usage of `send_email`